itest/riak_kv_sweeper_SUITE.erl

```diff
@@ -639,34 +662,17 @@ scheduler_estimated_keys_lock_fail_test(Config) ->
                          _LockResult = fail),
     meck_new_backend(TestCasePid, _NumKeys = 5000),
     SP = meck_new_sweep_particpant(sweep_observer_1, TestCasePid),
+    meck_new_visit_function(sweep_observer_1),
     SP1 = SP#sweep_participant{run_interval = 1},
     riak_kv_sweeper:add_sweep_participant(SP1),
-    riak_kv_sweeper:enable_sweep_scheduling(),
-    meck_new_visit_function(sweep_observer_1),
-    [ok = receive_msg({ok, successful_sweep, sweep_observer_1, I}, min_scheduler_response_time_msecs()) || I <- Indices],
+
+    sweep_all_indices(Indices),
+
     [true = meck:called(riak_kv_index_hashtree, get_lock, [I, estimate]) || I <- Indices],
     [false = meck:called(riak_kv_index_hashtree, estimate_keys, [I]) || I <- Indices],
     [false = meck:called(riak_kv_index_hashtree, release_lock, [I]) || I <- Indices],
     ok.
 
-scheduler_sweep_concurrency_test(_Config) ->
```

Comment:
A lot of the tests now run sweeps concurrently, since we bump up the sweep concurrency and spam a bunch of ticks in a row. I figured explicitly testing it is redundant now that it's being implicitly tested everywhere.